app/src/scss/modules/_defalt.scss

```diff
@@ -171,6 +174,62 @@ $padding--l: 30px;
     background-color: #fff;
   }
 
-  .Box--banner--center {
-      height: 250px;
+  .Box-form {
+    display: flex;
+    padding: $padding--l;
+    position: relative;
+    justify-content: center;
+  }
+
+  .form-box-1 {
+    color: #fff;
+    background-color: #000;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .form-box-2 {
+    background-color: #fff;
+    color: #000;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .form {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .form p {
+    display: flex;
+    justify-content: center;
+  }
+
+  .form-name {
+    display: flex;
+    justify-content: center;
+  }
+
+  .form-email {
+    display: flex;
+    justify-content: center;
+  }
+
+  .email-input {
+    width: 300px;
+  }
+
+
+  .Box-form-container {
+    display: flex;
+  }
+
+  .form-container-left {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .form-container-right {
+    display: flex;
+    flex-direction: column;
```

Comment:
Her kan du forenkle ting litt, da du ser at mange av disse klassene er helt like. For eksempel, mesteparten av form elementene dine inneolder display flex. introduserer du felles klasser for dette, krymper CSSen din en del. Du har allerede  laget .Box og .Box--center/right/left, du kan godt bruke noe lignende til form elementene. Eventuelt utvide .Box med en &--large og &--small der du setter stor og liten padding.